pipeline/upstream_listener.groovy

```diff
@@ -84,6 +84,11 @@ node("rhel-9-medium") {
                 job: "rhceph-upstream-listener",
                 parameters: [string(name: 'releaseName', value: "quincy")]
             ])
```

Comment:
@sunilkumarn417, IMO we need to include else if condition before build
else if ( upstreamVersion == "quincy" )

Reply:
I think we can start two jobs from main one.

Reply:
its working, https://jenkins.ceph.redhat.com/job/rhceph-upstream-listener/869/console